pydy/tests/test_system.py

```diff
@@ -395,3 +397,67 @@ def test_integrate(self):
         sys = System(self.kane, times=times)
         with testing.assert_raises(NotImplementedError):
             sys.generate_ode_function(generator='made-up')
+
+    def test_joint_system_init(self):
+        sys = System(method='joints')
```

Comment:
I was imagining that the `'JointMethod` would be equivalent in function as `KanesMethod` and `LagrangesMethod`.  Instead of a System class managing all the bodies and joints, the `JointMethod` class would do that. And we could leave our current system class as is. But then the question is where do we include the visualization stuff. I think adding viz decorators only makes sense on the System class, if we are trying to delineate purely symbolic classes from the classes that mix numerics and symbolics.

This is just a brain fart. I'm not sure it is a good idea yet, but I'll think about it this weekend some more. What do you think?